mne/viz/evoked.py

```diff
@@ -890,6 +896,13 @@ def plot_evoked_white(evoked, noise_cov, show=True):
                               scalings=None, rank=None, show=show)
 
 
+def _match_proj_type(proj, ch_names):
+    """see if proj should be counted"""
+    proj_ch_names = proj['data']['col_names']
+    select = any(kk in ch_names for kk in proj_ch_names)
+    return select
```

Comment:
This is because in the plot we display the rank by sensors, so you need to wee which projector is related to which sensor. See the tutorial, there you want to display rank for grad is 203 but 99 for mag, as we have 3 mag projectors. Same for EEG where you have 1 average reference and 1 bad etc. Ok?